### ISSUE TITLE
Class munkireport\processors\Processor located in ./app/processors/processor.php does not comply with psr-4 autoloading standard

### ISSUE THREAD

**Issue by GenoMyb** [munkireport/munkireport-php#1312]
When i do a
php composer.phar install --no-dev --no-suggest --optimize-autoloader
i get the below
Loading composer repositories with package information
Installing dependencies from lock file
Nothing to install or update
Generating optimized autoload files
Deprecation Notice: Class munkireport\processors\Processor located in ./app/processors/processor.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\models\MRModel located in ./app/models/mrmodel.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\models\Hash located in ./app/models/hash.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\models\Tablequery located in ./app/models/tablequery.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\models\Machine_group located in ./app/models/machine_group.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\models\Business_unit located in ./app/models/business_unit.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\models\Migration located in ./app/models/migration.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Auth located in ./app/controllers/auth.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Install located in ./app/controllers/install.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Settings located in ./app/controllers/settings.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Database located in ./app/controllers/database.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Error located in ./app/controllers/error.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Module located in ./app/controllers/module.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185
Deprecation Notice: Class munkireport\controller\Locale located in ./app/controllers/locale.php does not comply with psr-4 autoloading standard. It will not autoload anymore in Composer v1.11+. in phar:///usr/local/munkireport/composer.phar/src/Composer/Autoload/ClassMapGenerator.php:185

**Comment by jps3** [comment]
I am not well versed with PHP or the `composer` app, but I noticed those same error messages this morning upgrading to 5.3.1 and _after_ I had performed a `composer self-update` (which upgraded from 1.9.3 to 1.10.1).

I tried the `composer self-update --rollback` which was suggested in the output after the `self-update` to return to 1.9.3, and performed the `composer upgrade --no-dev` again. This time I did not see that error output.

Perhaps those warnings/errors are related to changes in `composer`?
